judge/admin/problem.py

```diff
@@ -248,3 +249,23 @@ def construct_change_message(self, request, form, *args, **kwargs):
         if form.cleaned_data.get('change_message'):
             return form.cleaned_data['change_message']
         return super(ProblemAdmin, self).construct_change_message(request, form, *args, **kwargs)
+
+
+class ProblemPointsVoteAdmin(admin.ModelAdmin):
```

Comment:
No comment right now, but a reminder that I should investigate of the behaviour of this more thoroughly when I pull and do a black box test.